firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/CrashlyticsController.java

```diff
@@ -1284,10 +1324,10 @@ private static void appendToProtoFile(File file, CodedOutputStreamWriteAction wr
     }
   }
 
-  private void writeBeginSession(final String sessionId, final Date startedAt) throws Exception {
+  private void writeBeginSession(final String sessionId, final long startedAtSeconds)
+      throws Exception {
     final String generator =
         String.format(Locale.US, GENERATOR_FORMAT, CrashlyticsCore.getVersion());
-    final long startedAtSeconds = startedAt.getTime() / 1000;
```

Comment:
I see, we always converted it to seconds, so that's why you're doing that here.

Reply:
:+1: